lua/nvim-treesitter/parsers.lua

```diff
@@ -91,6 +91,14 @@ list.agda = {
   maintainers = { "@Decodetalkers" },
 }
 
+list.apex = {
+  install_info = {
+    url = "https://github.com/aheber/tree-sitter-sfapex/tree/main/apex",
```

Comment:
This is still wrong, please check the `list.csv` for an example of a correct non-toplevel parser

Reply:
@lucario387 These three parsers worked seamlessly with local installation in my nvim. Do they need to be restructured to align with standards or?

@aheber Not sure what to be changed. 
Your call here. https://github.com/amaanq/tree-sitter-csv

Reply:
No need for his call, you need to use `location = "apex"` instead of declaring `tree/main/apex`

Reply:
Example with `csv` parser

```lua
list.csv = {
  install_info = {
    url = "https://github.com/amaanq/tree-sitter-csv",
    files = { "src/parser.c" },
    location = "csv",
  },
  maintainers = { "@amaanq" },
}
```

Reply:
Thanks, done